pandas/tests/io/test_clipboard.py

```diff
@@ -124,6 +145,34 @@ def test_read_clipboard_infer_excel(self):
 
         tm.assert_frame_equal(res, exp)
 
+    def test_excel_clipboard_tabs(self):
+        for dt in self.data_types:
+            for sep in ['\t', None, 'default', ',', '|']:
+                for excel in [True, None, 'default', False]:
+                    # Function default should be to to produce tab delimited
+                    kwargs = {}
+                    if excel != 'default':
+                        kwargs['excel'] = excel
+                    if sep != 'default':
+                        kwargs['sep'] = sep
+                    data = self.data[dt]
+                    data.to_clipboard(**kwargs)
+                    if sep in ['\t', None, 'default'] and excel is not False:
+                        # Expect tab delimited
+                        result = read_clipboard(sep='\t', index_col=0)
+                        tm.assert_frame_equal(data, result, check_dtype=False)
+                        assert clipboard_get().count('\t') > 0
```

Comment:
Instead of counting the number of tabs can you construct the expected output and match exactly against that?